### ISSUE TITLE
Doesn't work with new version of `pex-context`

### ISSUE THREAD

**Issue by mhintz** [pex-gl/pex-sys#19]
Hello! I'm taking a look at some pex stuff and I found that the examples seem to be broken by the newer interfaces in pex-camera and pex-context. In particular, pex-sys no longer seems to work because it expects the old API for pex-context.

Do you have a development branch of pex-sys which uses the new pex-context API so I could try it out?

Thanks!

**Comment by vorg** [comment]
Hi @mhintz first sorry to cause the trouble. Unfortunately there is not new version of pex-sys in the works yet. Pex context is able to create a Plask window or HTML canvas by itself now. Saying that the issue of e.g. unified mouse handling in new pex-context apps comes more and more often so pex-sys is a good candidate for update. Could you say what you are working on and which pex-sys classes are crucial?

**Comment by mhintz** [comment]
Hi Marcin! Sorry I only just saw this message. I was actually confused by the new API so I didn't realize that pex-sys was kind of outdated by the new API for pex-context. I checked out the examples and the source code for pex-context and was able to figure things out :)

I'm just messing around with pex trying to find a nice WebGL library to fiddle with. In case I find anything that could be updated, I'll submit a PR :)
